### PR TITLE
wepr

### PR DESCRIPTION
**Pull request by sagar4ahirrao**
Wer sdff

### CODE REVIEW

**Comment by sagar4ahirrao** [comment]
rerwe

**Comment by sagar4ahirrao** [comment]
dfdsfdfsdggf

**Comment by sagarvi-dev** [comment]
comment

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
+  #helloogpr
```
Comment (sagarvi-dev) [inline app/controllers/application_controller.rb:4]: dddds

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -50,3 +50,4 @@ def set_post
     @post = Post.find(params[:id])
   end
```
Comment (sagarvi-dev) [inline app/controllers/posts_controller.rb:51]: postfdfd

**Pull request merged**